docs/userguide/main.md

```diff
@@ -1036,6 +1036,40 @@ spec:
   description: Sample tenant
 ```
 
+
+### PodDisruptionBudget
+
+
+The PDB (Pod Disruption Budget) is a Kubernetes resource that helps ensure the high availability of applications by defining the acceptable disruption level during maintenance or unexpected events.
+It specifies the minimum number of pods that must remain available to maintain the desired level of service.
+The PDB definition is unique for every nodePool.
+Please be aware that you must provide `MinAvailable` and `MaxUnavailable` configurations in order to use PDB feature.
```

Comment:
```suggestion
You must provide either `minAvailable` or `maxUnavailable` to configure PDB, but noth both.
```

Reply:
This does not make clear to the user that only one of the fields can be set. Also: field names must start lowercase.